lib/avro-patches/logical_types/schema.rb

```diff
@@ -57,8 +57,8 @@ def self.real_parse(json_obj, names=nil, default_namespace=nil)
   end
 
   # Determine if a ruby datum is an instance of a schema
-  def self.validate(expected_schema, logical_datum, recursive = true, encoded = false)
-    Avro::SchemaValidator.validate!(expected_schema, logical_datum, recursive, encoded)
+  def self.validate(expected_schema, logical_datum, options = { recursive: true, encoded: false })
```

Comment:
I wasn't relying on it for defaulting, more for documentation. 

If `:encoded` is not specified, it will be `nil` and interpreted as falsey anyway.

Handling for defaulting `:recursive` is pushed down into `SchemaValidator.validate!`.